static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -110,6 +118,8 @@ export type UnsavedMetricRule = {
   monitorWindow?: number | null;
   owner?: string | null;
   queryType?: MEPAlertsQueryType | null;
+  seasonality?: string | null;
```

Comment:
could this become an enum with `auto` and any other types? (i'm guessing we can only support a limited number of types of seasonality)